src/platform/silabs/wifi/rs911x/WifiInterface.cpp

```diff
@@ -966,8 +933,26 @@ int32_t wfx_rsi_send_data(void * p, uint16_t len)
  * @return  SL_STATUS_OK if successful,
  *          SL_STATUS_FAIL otherwise
  ***********************************************************************/
-sl_status_t wfx_power_save(void) // TODO : Figure out why the extern C is necessary
```

Comment:
I take it we figured it out?

Reply:
yep! i forgot to remove the comment in CSA when i fixed the problem.